src/api/public/apidocs-new/paths/build_project_repository_architecture_package_file.yaml

```diff
@@ -37,3 +37,72 @@ get:
             details: "404 project 'home:Admin' has no repository '4'"
   tags:
     - Build
+
+put:
+  summary: This endpoint updates a specific artifact file contents
+  description: |
+    Given a project, repository, architecture and package, update the given file's content.
+  operationId: putBuildProjectRepositoryArchitecturePackageFile
```

Comment:
From the (original documentation)[https://build.opensuse.org/apidocs/index.html#108] this can only be done as ADMIN. Could you check this?

Reply:
Confirmed. Only Admin can do this